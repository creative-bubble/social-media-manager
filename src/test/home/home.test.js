import { renderWith } from '../helper/helper';

describe('Suite === Home Page', () => {
  describe('Rendering', ()  => {
    it('Render whole page of Home', () => {
      const { asFragment } = renderWith('/');
      expect(asFragment()).toMatchSnapshot('Home-Page');
    });
  });
});

