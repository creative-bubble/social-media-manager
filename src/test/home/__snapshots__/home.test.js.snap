// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Suite === Home Page Rendering Render whole page of Home: Home-Page 1`] = `
<DocumentFragment>
  <div
    class="bg-gray-900"
  >
    <div
      class="fixed w-full h-10vh pt-4 text-gray-300 bg-gray-900 z-10 flex flex-row sm:justify-center sm:relative sm:w-9/12 sm:mx-auto"
      id="naver"
    >
      <div
        class="ml-2 sm:ml-0 text-xs md:text-sm lg:text-lg sm:absolute sm:left-0"
      >
        #
        <span
          class="text-purple-600"
        >
          Social
        </span>
         Media Manager
      </div>
      <button
        class="absolute right-0 top-0 mt-4 mx-4 z-10 sm:hidden border border-blue-500 text-blue-500 w-8 h-8 hover:bg-blue-500 hover:text-gray-200 focus:outline-none"
      >
        <i
          aria-hidden="true"
          class="fa fa-bars"
        />
      </button>
      <div
        class="sm:static sm:flex sm:pt-0"
        id="nav-container"
      >
        <nav
          class="text-gray-500 ml-2 sm:ml-0"
        >
          <ul
            class="text-xs md:text-base sm:grid sm:grid-flow-col sm:gap-8"
          >
            <li
              class="border-b border-gray-900 cursor-pointer rounded hover:bg-gray-800 sm:hover:bg-transparent sm:hover:border-green-500 sm:rounded-none"
            >
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li
              class="mt-1 sm:mt-0 border-b border-gray-900 cursor-pointer rounded hover:bg-gray-800 sm:hover:bg-transparent sm:hover:border-green-500 sm:rounded-none"
            >
              <a
                href="/group"
              >
                Media
              </a>
            </li>
            <li
              class="mt-1 sm:mt-0  border-b border-gray-900 cursor-pointer rounded hover:bg-gray-800 sm:hover:bg-transparent sm:hover:border-green-500 sm:rounded-none"
            >
              <a
                href="/about"
              >
                About
              </a>
            </li>
          </ul>
        </nav>
      </div>
      <div
        class="w-12 pl-2 sm:pl-0 sm:absolute sm:right-0"
      >
        <button
          class="w-full border border-blue-500 bg-transparent text-blue-500 rounded-sm focus:outline-none hover:bg-blue-500 hover:text-gray-300"
        >
          USER
        </button>
      </div>
    </div>
    <div
      class="block sm:hidden w-full h-10vh"
    />
  </div>
  <div
    class="min-h-screen"
    id="main-content"
  >
    <section
      class="container-xl bg-gray-900"
    >
      <div
        class="h-88 "
      >
        <div
          class="w-9/12 mx-auto relative"
        >
          <div
            class="text-3xl sm:text-4xl font-Bebas pt-12"
          >
            <div
              class="pt-12 flex flex-col  text-gray-200 w-full text-center lg:text-left"
            >
              <span>
                Easy Distributed Post
              </span>
              <span>
                On Multimedia Platforms
              </span>
              <div>
                <button
                  class="w-8/12 mt-10 lg:w-4/12 border border-blue-500 text-blue-500 hover:bg-blue-500 hover:text-gray-200"
                >
                  GET STARTED
                </button>
              </div>
            </div>
          </div>
          <div
            class="w-5/12 h-76 absolute right-0 top-0 hidden lg:block md:mt-10"
          >
            <img
              alt="media2"
              class="object-contain"
              src="[object Object]"
            />
          </div>
        </div>
      </div>
       
    </section>
    <section
      class="container-xl"
    >
      <div
        class="mt-1 md:mt-8 lg:mt-32"
      >
        <div
          class="w-9/12 mx-auto"
        >
          <div
            class="flex justify-center pt-12 sm:pt-0"
          >
            <h2
              class="font-Bebas text-2xl text-purple-900"
            >
              Our Services
            </h2>
          </div>
          <div
            class="mt-2 flex flex-col sm:flex-row sm:justify-around sm:mt-6 "
          >
            <div
              class="w-12/12 mt-6 sm:w-3/12 sm:h-88"
            >
              <div
                class="w-full relative sm:h-40"
              >
                <img
                  alt="media1"
                  class="sm:absolute h-full w-full object-contain"
                  src="[object Object]"
                />
              </div>
              <div
                class="mt-2 text-md sm:text-md sm:mt-10 text-purple-900 text-center font-Bebas"
              >
                <h3>
                  Fast Posting
                </h3>
                <span>
                  Simple and Easy to use for any Devices
                </span>
                <span>
                  with a simple click
                </span>
              </div>
            </div>
            <div
              class="w-12/12 mt-6 sm:w-3/12 sm:h-88"
            >
              <div
                class="w-full relative sm:h-40"
              >
                <img
                  alt="media1"
                  class="sm:absolute h-full w-full object-contain"
                  src="[object Object]"
                />
              </div>
              <div
                class="mt-2 text-md sm:text-md sm:mt-10 text-purple-900 text-center font-Bebas"
              >
                <h3>
                  Fast Posting
                </h3>
                <span>
                  Simple and Easy to use for any Devices
                </span>
                <span>
                  with a simple click
                </span>
              </div>
            </div>
            <div
              class="w-12/12 mt-6 sm:w-3/12 sm:h-88"
            >
              <div
                class="w-full relative sm:h-40"
              >
                <img
                  alt="media1"
                  class="sm:absolute h-full w-full object-contain"
                  src="[object Object]"
                />
              </div>
              <div
                class="mt-2 text-md sm:text-md sm:mt-10 text-purple-900 text-center font-Bebas"
              >
                <h3>
                  Fast Posting
                </h3>
                <span>
                  Simple and Easy to use for any Devices
                </span>
                <span>
                  with a simple click
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <div
      class="container-xl mt-6 pb-20 sm:mt-0"
    >
      <div
        class="relative sm:w-9/12 mx-auto flex flex-col md:flex-row"
      >
        <br
          class="sm:hidden"
        />
        <div
          class=" w-full md:w-6/12"
        >
          <div
            class="w-9/12 mx-auto"
          >
            <div
              class="relative h-16 sm:h-20 flex justify-center sm:justify-start"
            >
              <h2
                class="absolute mt-6 font-semibold text-blue-500"
              >
                CONTACT US
              </h2>
            </div>
            <p
              class="text-xs font-semibold text-blue-500"
            >
              Drop a message
            </p>
            <div>
              <input
                class="w-full mt-3 rounded-sm border border-gray-400 focus:outline-none focus:border-blue-500"
              />
            </div>
            <div>
              <input
                class="w-full mt-3 rounded-sm border border-gray-400 focus:outline-none focus:border-blue-500"
              />
            </div>
            <div>
              <input
                class="w-full mt-3 rounded-sm border border-gray-400 focus:outline-none focus:border-blue-500"
              />
            </div>
            <div>
              <input
                class="w-full mt-3 rounded-sm border border-gray-400 focus:outline-none focus:border-blue-500"
              />
            </div>
            <textarea
              class="w-full h-40 mt-3 rounded-sm border border-gray-400 focus:outline-none focus:border-blue-500"
              placeholder="Message*"
            />
            <button
              class="w-full h-8 mt-4 text-xs rounded-full bg-blue-500 text-gray-200 tracking-wider focus:outline-none active:bg-blue-400 "
            >
              SUBMIT
            </button>
          </div>
        </div>
        <div
          class="absolute bottom-0 right-0 w-6/12 h-64 mx-auto hidden md:block"
        >
          <img
            alt="media1"
            class="h-full w-full object-contain"
            src="[object Object]"
          />
        </div>
      </div>
    </div>
  </div>
  <footer
    class="container-xl w-full h-48 bg-gray-900"
  >
    <div
      class="w-9/12 mx-auto"
    >
      <div
        class="relative h-48"
      >
        <div
          class="absolute w-full h-32 mt-0 pt-6 text-xs text-gray-200 lg:w-8/12 lg:h-40 lg:text-md leading-loose"
        >
          <div
            class="flex flex-row"
          >
            <div>
              <span
                class="font-bold"
              >
                |
              </span>
               
              <span
                class="font-bold"
              >
                 Contact Info
              </span>
            </div>
            <div>
              <i
                aria-hidden="true"
                class="fa fa-instagram text-green-500 px-2"
              />
              <i
                aria-hidden="true"
                class="fa fa-facebook-official text-green-500 px-2"
              />
              <i
                aria-hidden="true"
                class="fa fa-twitter-square text-green-500 px-2"
              />
              <i
                aria-hidden="true"
                class="fa fa-twitch text-green-500 px-2"
              />
            </div>
          </div>
          <div
            class="flex flex-col"
          >
            <div>
              <div>
                <i
                  aria-hidden="true"
                  class="fa fa fa-envelope text-green-500 pr-1"
                />
                 media@gmail.com
              </div>
              <div>
                <i
                  aria-hidden="true"
                  class="fa fa-phone-square text-green-500 pr-1"
                />
                 +63(999)037-1921
              </div>
              <div>
                <i
                  aria-hidden="true"
                  class="fa fa fa-compass text-green-500 pr-2"
                />
                 Metro Manila
              </div>
            </div>
            <div>
              <div>
                <i
                  aria-hidden="true"
                  class="fa fa-facebook-official text-green-500 pr-1"
                />
                 Social Media
              </div>
              <div>
                <i
                  aria-hidden="true"
                  class="fa fa-twitter-square text-green-500 pr-1"
                />
                 @Social Media
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
